src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -100,18 +104,19 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         // No validation
-        internal CSharpParseOptions(
+        private CSharpParseOptions(
```

Comment:
>private CSharpParseOptions( [](start = 8, length = 27)

It looks like there is only one use site for this overload and it is from another constructor. I think we should simply inline the code and remove this constructor, otherwise we might make the same mistake again.